applications/GeoMechanicsApplication/tests/cpp_tests/test_utilities/spy_condition.h

```diff
@@ -25,9 +26,14 @@ class SpyCondition : public Condition
     void InitializeNonLinearIteration(const ProcessInfo& rCurrentProcessInfo) override;
     void FinalizeNonLinearIteration(const ProcessInfo& rCurrentProcessInfo) override;
 
-    void EquationIdVector(EquationIdVectorType& rResult, const ProcessInfo& rCurrentProcessInfo) const override;
-
-    void GetDofList(DofsVectorType& rElementalDofList, const ProcessInfo& rCurrentProcessInfo) const override;
+    MOCK_METHOD(void,
+                EquationIdVector,
+                (EquationIdVectorType & rResult, const ProcessInfo& rCurrentProcessInfo),
+                (const, override));
+    MOCK_METHOD(void,
+                GetDofList,
+                (DofsVectorType & rElementalDofList, const ProcessInfo& rCurrentProcessInfo),
+                (const, override));
```

Comment:
This can be compacted a bit:
```suggestion
    MOCK_METHOD(void,
                EquationIdVector,
                (EquationIdVectorType & , const ProcessInfo& ),
                (const, override));
    MOCK_METHOD(void,
                GetDofList,
                (DofsVectorType & , const ProcessInfo& ),
                (const, override));
```